### PR TITLE
Bugfix for getIcon indexer and conflicting "image" id

### PR DESCRIPTION
**Pull request by frapell**
This PR fixes https://github.com/plone/Products.CMFPlone/issues/3916

### CODE REVIEW

**Comment by frapell** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/tests/testCatalogTool.py [hunk 1]
```diff
@@ -1414,6 +1415,21 @@ def test_uuid(self):
         self.assertTrue(wrapped.UID)
         self.assertTrue(uuid == wrapped.UID)
 
+    def test_getIcon(self):
+        from Products.CMFPlone.CatalogTool import getIcon
```
Comment (davisagli) [inline Products/CMFPlone/tests/testCatalogTool.py:1419]: getIcon here vs get_icon below?

**Review by davisagli — APPROVED** [review]
Looks okay to me, except for one small question.

Could you open a PR to the master branch too, so the change doesn't get stuck in 6.0 only?

File: Products/CMFPlone/tests/testCatalogTool.py (hunk 1 above)
Comment (frapell) [reply Products/CMFPlone/tests/testCatalogTool.py:1419]: yeah, getIcon is a `plone.indexer.delegate.DelegatingIndexerFactory` and by doing `getIcon.callable` I get the actual function to test.
This is the same way it is done above, with `getObjSize`

Comment (davisagli) [reply Products/CMFPlone/tests/testCatalogTool.py:1419]: @frapell yep sorry, I overlooked that line

**Pull request merged**